src/Manager.cpp

```diff
@@ -32,11 +32,11 @@ BDD_ID Manager::createVar(const std::string &label) {
     return 0;
 }
 
-void Manager::AddNode(BDD_ID a, BDD_ID b, BDD_ID c, BDD_ID d, std::string &e){
-     BDD_node new_var(a,b,c,d, (std::string&)e);
-     unique_table.push_back(new_var);
-
+void Manager::addNode(BDD_ID a, BDD_ID b, BDD_ID c, BDD_ID d, std::string &e){
```

Comment:
I think we could have put this inside the function that is calling it and for testing purposes used the SetUp() method from GTest.